tests/src/eth/migration.test.ts

```diff
@@ -105,4 +125,83 @@ describe('EVM Migrations', () => {
       expect(await contract.methods.get(i).call()).to.be.equal(i.toString());
     }
   });
+  itEth('Fake collection creation on substrate side', async ({helper}) => {
+    const txInsertEvents = helper.constructApiCall('api.tx.evmMigration.insertEvents', [[
+      encodeEvent(helper.api!, 'Common', 'common', 'CollectionCreated', [
```

Comment:
```suggestion
      encodeEvent(helper.getApi(), 'Common', 'common', 'CollectionCreated', [
```

И так же в других местах